proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -45,119 +44,126 @@ If the SDL app or HU cannot use this function, it will perform the same operatio
-+        <description>Set whether or not this app requires the use of an audio streaming output device.</description>
-+    </param>
-</struct>
+  <struct name="DeviceInfo">
+      <param name="name" type="String" mandatory="true">
+          <description>The name of the device connected.</description>
+      </param>
+      <param name="id" type="String" mandatory="true">
+          <description>The ID of the device connectedi: either hash of device's USB serial number(in case of USB connection) or has of device's MAC address(in case of BlueTooth or WIFI connection</description>
+      </param>
+      <param name="transportType" type="Common.TransportType" mandatory="false">
+         <description>The transport type the named-app's-device is connected over HU(BlueTooth, USB or WiFi). It must be provided in OnAppRegistered and in UpdateDeviceList</description>
+      </param>
+      <param name="isSDLAllowed" type="Boolean" mandatory="false">
+          <description>Sent by SDL in UpdateDeviceList. 'true' - if device is allowed for PolicyTable Exchange; 'false' - if device is NOT allowed for PolicyTable Exchange </description>
+      </param>
++     <param name="bluetoothDeviceAddress" type="String" mandatory="false">
++         <description>Device BT Address</description>
++     </param>
+  </struct>
+  ...
+  <struct name="HMIApplication">
+  ...
+      <param name="deviceInfo" type="Common.DeviceInfo" mandatory="true">
+          <description>The ID, serial number, and transport type by which the named app's device is connected to HU.</description>
+      </param>
+  ...
+      <param name="cloudConnectionStatus" type="Common.CloudConnectionStatus" mandatory="false"></param>
++     <param name="requiresAudioSupport" type="Boolean" mandatory="false">
++         <description>Set whether or not this app requires the use of an audio streaming output device.</description>
++     </param>
+  </struct>
 ```
 
+### MediaStreamingStatus class
+
+The library supports the new version of the protocol specification. However the app needs to use `MediaStreamingStatus` class before registering if the IVI system uses the earlier version.
+With the changes of the flow, it is necessary to do refactoring the library to move the logic of `MediaStreamingStatus` class.
+
 ### Launch the app
 
-By using this function, the HU is able to perform  BT automatic connection based on the `bluetoothDeviceAddress` which is notified when the app that requests audio (`requiresAudioSupport` is set to true) is launched. 
+By using this function, the HU is able to perform  BT automatic connection based on the `bluetoothDeviceAddress` which is notified when the app that requests audio (`requiresAudioSupport` is set to true) is launched.
 It can also display a message that prompts BT connection to the user.
 
+Note: While the HMI is connecting with the Bluetooth mobile device of the app that is set up `requiresAudioSupport=true`, preventions from activations of the app are needed.
```

Comment:
I think this should be re-worded to be more aligned with [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/941#issuecomment-644911918)
```suggestion
Note: The HMI will be responsible for preventing app activation while the HMI is in the process of connecting to the mobile device's BT for an app that has `requiresAudioSupport=true`. For example, the HMI could show a loading icon for that app, or display an error message to the user upon activation.
```